@navikt/core/tokens/darkside/figma/figma-config.types.ts

```diff
@@ -0,0 +1,28 @@
+import { StyleDictionaryToken, TokenTypes } from "../util";
+
+export type FigmaToken = Omit<StyleDictionaryToken<TokenTypes>, "value"> & {
+  name: string;
+  alias?: string;
+  code: {
+    web: string;
+  };
+  value: string | number;
+  figmaType: VariableResolvedDataType;
+  scopes: VariableScope[];
+};
+
+type FigmaConfigEntry = {
+  name: string;
+  hideFromPublishing: boolean;
+  token: FigmaToken[];
```

Comment:
Unless this is required by Figma, consider using plural since it's an array
```suggestion
  tokens: FigmaToken[];
```

Reply:
Done